synchronizer/synchronizer.go

```diff
@@ -1361,7 +1382,7 @@ func (s *ClientSynchronizer) processTrustedBatch(trustedBatch *types.Batch, dbTx
 		// Update batchL2Data
 		err := s.state.UpdateBatchL2Data(s.ctx, batches[0].BatchNumber, trustedBatchL2Data, dbTx)
 		if err != nil {
-			log.Errorf("error opening batch %d", trustedBatch.Number)
+			log.Errorf("error opening batch %v", trustedBatch.Number)
```

Comment:
Why we use here %v and in lines 1346 and 1349 we use %d with the uint64 casting? I think we can use the same "solution" in all the places

Reply:
In this case the type it is a custom one called "types.ArgUint64". Not exactly uint64.  %v is the generic so will work always, thats why I decided to use %v